src/sockperf.cpp

```diff
@@ -1718,7 +1718,27 @@ static int resolve_sockaddr(const char *host, const char *port, int sock_type,
     int res = getaddrinfo(host, port, &hints, &result);
     if (res == 0) {
         get_socket_address(result, addr, addr_len);
+    } else if (res == EAI_NONAME && host != NULL) { // Unix Domain Socket name
+        log_dbg("provided path for Unix Domain Socket is %s\n", host);
+        std::string path = host;
+        if (!(path.rfind("/", 0) == 0)) { // limit to prefix
```

Comment:
1. `path.size() > 0 && path[0] == '/'`
2. It's better to move this check before `getaddrinfo` invocation because getaddrinfo on file name will always fail.

Reply:
1) will be changed 
2) if I move the condition before getaddrinfo, so in case we have path "tmp/test" and not "/tmp/test" 
it will not enter to first condition - since it doesn't start with '/', and will fail on the second.